ui/components/app/signature-request-original/signature-request-original.component.js

```diff
@@ -127,13 +127,13 @@ export default class SignatureRequestOriginal extends Component {
       msgParams: { data },
     } = txData;
 
-    if (type === MESSAGE_TYPE.PERSONAL_SIGN) {
+    if (type === ApprovalType.PersonalSign) {
```

Comment:
All `MESSAGE_TYPE` here as it's the transaction type which corresponds to the JSON-RPC method.